Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -705,6 +705,7 @@ quoteListModelViewState state = { source: state.data.source
                             , selectedQuote: state.props.selectedQuote
                             , autoSelecting: state.props.autoSelecting
                             , searchExpire: state.props.searchExpire
+                            , showProgress : if ( (DA.null state.data.quoteListModelState) && isLocalStageOn FindingQuotes && state.props.currentStage /= TryAgain) then true else false
```

Comment:
We can just have `(DA.null state.data.quoteListModelState) && isLocalStageOn FindingQuotes && state.props.currentStage /= TryAgain` no need to have if else case?

Reply:
yes